taskt/Core/Automation/Commands/Outlook Commands/GetOutlookEmailsCommand.cs

```diff
@@ -1,109 +1,115 @@
+        [Remarks("")]
         public string v_GetUnreadOnly { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Mark emails as read")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Yes")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("No")]
-        [Attributes.PropertyAttributes.InputSpecification("Specify whether to retrieve unread email messages only")]
-        [Attributes.PropertyAttributes.SampleUsage("Select **Yes** or **No**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Mark As Read")]
+        [PropertyUISelectionOption("Yes")]
+        [PropertyUISelectionOption("No")]
+        [InputSpecification("Specify whether to mark retrieved emails as read.")]
+        [SampleUsage("")]
+        [Remarks("")]
         public string v_MarkAsRead { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the output directory for the messages")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter or Select the path to the directory.")]
-        [Attributes.PropertyAttributes.SampleUsage("C:\\temp\\myfolder or [vTextFolderPath]")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_MessageDirectory { get; set; }
+        [PropertyDescription("Save MailItems and Attachments")]
+        [PropertyUISelectionOption("Yes")]
+        [PropertyUISelectionOption("No")]
+        [InputSpecification("Specify whether to save the email attachments to a local directory.")]
+        [SampleUsage("")]
+        [Remarks("")]
+        public string v_SaveMessagesAndAttachments { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Assign MailItem List to variable")]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable from the variable list")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required to pre-define your variables, however, it is highly recommended.")]
-        public string v_userVariableName { get; set; }
+        [PropertyDescription("Output MailItem Directory")]   
+        [InputSpecification("Enter or Select the path of the directory to store the messages in.")]
+        [SampleUsage(@"C:\temp\myfolder || {vFolderPath} || {ProjectPath}\myFolder")]
+        [Remarks("This input is optional and will only be used if *SaveMailItems and Attachments* is set to *Yes*.")]
```

Comment:
Please add a space *Save<SPACE>MailItems and Attachments* to refer to the above property description.

Reply:
done